static/app/components/events/interfaces/frame/utils.spec.tsx

```diff
@@ -23,3 +27,104 @@ describe('hasFileExtension', () => {
     expect(hasFileExtension('example@.py')).toBe(true); // Special characters
   });
 });
+
+describe('shouldDisplayAbsPathInTitle', () => {
+  it('returns false when absPath is from event subdomain', () => {
```

Comment:
shouldn't this be `true` or the check `toBe(false)`?
```suggestion
  it('returns true when absPath is from event subdomain', () => {
```

Reply:
Thinking behind this test was: If my sources are hosted on the same  or sub-domain, I do not want to show the url by default.